expression/builtin.go

```
@@ -171,6 +171,10 @@ func (b *baseBuiltinFunc) getArgs() []Expression {
 	return b.args
 }
 
+func (b *baseBuiltinFunc) vecEval(input *chunk.Chunk, result *chunk.Column) error {
+	panic("baseBuiltinFunc.vecEval() should never be called")
```

please return an error.

---

How about format the error message like this:
```
baseBuiltinFunc.vecEval() should never be called, please contact the TiDB team for help
```

---

All comments above are fixed and all evaluate function `EvalXXType` return error instead of resulting in a panic now.